BraveRewardsUI/Welcome/WelcomeView.swift

```diff
@@ -13,6 +13,13 @@ extension WelcomeViewController {
       $0.contentEdgeInsets = UIEdgeInsets(top: 8, left: 12, bottom: 8, right: 12)
     }
     
+    let termsOfServiceLabel = LinkLabel().then {
+      $0.font = .systemFont(ofSize: 12.0)
+      $0.textColor = Colors.grey900
+      $0.textAlignment = .center
+      $0.text = Strings.DisclaimerInformation
```

Comment:
Wrong string here, there's a variant of the original message for the welcome screen:

`By clicking ‘Yes, I'm in!’, you indicate that you have read and agree to the Terms of Service and Privacy Policy.`